FreeRTOS/FreeRTOS-Plus-TCP
# Update RX sequence number after refusing TCP FIN packet.

Update RX sequence number after refusing TCP FIN packet.

Description
-----------

As FreeRTOS Forum [TCP Refusing FIN](https://forums.freertos.org/t/tcp-refusing-fin/21553) mentioned, the expected RX sequence is not well handled in the stack when refusing FIN. Update the RX sequence number when hitting this scenario.

Test Steps
-----------

Refer to [this forum post](https://forums.freertos.org/t/tcp-refusing-fin/21553/6) for the test result.
1. Create TCP connection.
2. Keep sending packets.
3. Receiving FIN packet.

Checklist:
----------


- [] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
